app/component/map/Map.js

```diff
@@ -283,7 +283,7 @@ export default class Map extends React.Component {
     };
 
     return (
-      <div aria-hidden="true">
+      <div>
```

Comment:
This change makes everything inside the map visible to screen readers. That must not happen - only the control buttons should be read.